test/system/maintenance_tasks/runs_test.rb

```diff
@@ -60,16 +60,42 @@ class RunsTest < ApplicationSystemTestCase
       MaintenanceTasks.metadata = nil
     end
 
-    test "run a CSV Task" do
-      visit maintenance_tasks_path
+    test "run a CSV Task without direct upload" do
+      with_direct_upload(false) do
+        visit maintenance_tasks_path
 
-      click_on("Maintenance::ImportPostsTask")
-      attach_file("csv_file", "test/fixtures/files/sample.csv")
-      click_on "Run"
+        click_on("Maintenance::ImportPostsTask")
 
-      assert_title "Maintenance::ImportPostsTask"
-      assert_text "Enqueued"
-      assert_text "Waiting to start."
+        assert_selector("input[type=file]") do |input|
+          assert_nil(input["data-direct-upload-url"])
+        end
+
+        attach_file("csv_file", "test/fixtures/files/sample.csv")
+        click_on "Run"
+
+        assert_title "Maintenance::ImportPostsTask"
+        assert_text "Enqueued"
+        assert_text "Waiting to start."
+      end
+    end
+
+    test "run a CSV Task with direct upload" do
+      with_direct_upload(true) do
+        visit maintenance_tasks_path
+
+        click_on("Maintenance::ImportPostsTask")
+
+        assert_selector("input[type=file]") do |input|
+          assert_equal(input["data-direct-upload-url"], rails_direct_uploads_url)
+        end
```

Comment:
Those tests are failing on Rails `<= 7.0`.

Not sure why, the usage is supposed to be the same: https://guides.rubyonrails.org/v6.1/active_storage_overview.html#usage

I could use some help 😅